src/us/kbase/auth2/lib/Password.java

```diff
@@ -1,5 +1,13 @@
 package us.kbase.auth2.lib;
 
+
+import com.nulabinc.zxcvbn.Zxcvbn;
+import com.nulabinc.zxcvbn.Strength;
+
+import org.bouncycastle.util.Arrays;
```

Comment:
Why not just use the built in Arrays.equal()? https://docs.oracle.com/javase/7/docs/api/java/util/Arrays.html#equals%28char[],%20char[]%29